src/Core/src/Handlers/Label/LabelHandler.cs

```diff
@@ -4,6 +4,9 @@ public partial class LabelHandler
 	{
 		public static PropertyMapper<ILabel, LabelHandler> LabelMapper = new PropertyMapper<ILabel, LabelHandler>(ViewHandler.ViewMapper)
 		{
+#if __IOS__ || IOS
+			[nameof(ICheckBox.Background)] = MapBackground,
```

Comment:
Labels on iOS are special in that they can't actually have a layer set. In Forms this "worked" because there was specific logic to check for labels and then apply the brush to the renderer view and not the label inside. Here we use custom logic on `MauiLabel`.